### ISSUE TITLE
Missing RenderHookResult type

### ISSUE THREAD

**Issue by vbfox** [testing-library/react-hooks-testing-library#536]
- `react-hooks-testing-library` version: 4.0.0

### Relevant code or config:

```js
import { RenderHookResult } from '@testing-library/react-hooks';
```

### What you did:

I Updated a TypeScript project that was using 3.x & that used `RenderHookResult` to 4.0.0

### What happened:

The type doesn't exists anymore

### Reproduction:

Any code importing the type should fail to compile

### Problem description:

Same problem as https://github.com/testing-library/react-hooks-testing-library/issues/534 but on the result type instead of the option parameter.

### Suggested solution:

Add the interface back to the new typescript-rewritten version

**Comment by joshuaellis** [comment]
this is fixed & released in `v5.0.0-beta.9`. However, you are welcome to submit a PR fixing the export issue.

**Comment by joshuaellis** [comment]
This is released in v5.0.0
